README.md

```diff
@@ -95,6 +95,14 @@ the complete filename), e.g. `["sha1", "md5"]`.
 
   A list of a tag files that must be included in a conformant Bag. Entries are full path names relative to the Bag base directory. As per the [BagIt Spec](http://tools.ietf.org/html/draft-kunze-bagit-08), these tag files need not be listed in tag manifiest files. `Tag-Files-Required` SHOULD NOT include `bag-info.txt` (which is always required), nor any required manifest files, which instead are required by `Manifests-Required` and `Tag-Manifests-Required`.
 
+  Every file in `Tag-Files-Required` must also be present in `Tag-Files-Allowed`.
+
+10. `Tag-Files-Allowed`: LIST
+
+  A list of tag files that may be included in a conformant Bag. Entries are full path names relative to the Bag base directory. As per the [BagIt Spec](http://tools.ietf.org/html/draft-kunze-bagit-08), these tag files need not be listed in tag manifiest files. `Tag-Files-Required` SHOULD NOT include `bag-info.txt` (which is always required), nor any required manifest files, which instead are required by `Manifests-Required` and `Tag-Manifests-Required`.
```

Comment:
Sure. BTW, the purl.org link (`http://purl.org/net/bagit`) points to `https://tools.ietf.org/html/draft-kunze-bagit-17`, not `https://tools.ietf.org/html/rfc8493`.

Reply:
And perhaps it would help to restructure the README so that all the relevant sections are preceded by headers, so Github assigns them an ID, and they can be referred to?

```markdown
7. `Accept-BagIt-Version`: LIST

| A list of BagIt version numbers that will be accepted. At least one version is required.
```

-->

```markdown
### Accept-BagIt-Version

`Accep-Bagit-Version: LIST`

| A list of BagIt version numbers that will be accepted. At least one version is required.
```

Would allow linking to `https://github.com/bagit-profiles/bagit-profiles/#accept-bagit-version`. Alternatively, one could add link anchors, e.g.

```markdown
<a name="accept-bagit-version"/>
7. `Accept-BagIt-Version`: LIST

| A list of BagIt version numbers that will be accepted. At least one version is required.
```